std_msgs/String imageDir
dataset/SURF config
---
std_msgs/String leftFoundDir
std_msgs/Int32 nleft

 
 
